test_gh_config.yaml

```diff
@@ -2,8 +2,8 @@
 # 1 profile loaded with default speeds is defined for each vehicle type
 
 graphhopper:
-  datareader.file: test-data/kansas-city-extract-mini.osm.pbf
-  gtfs.file: test-data/mini_kc_gtfs.tar
+  datareader.file: TEST_OSM
```

Comment:
errr I'm not sure - I think the only reason we need curly braces in the python code is because python's `format_map()` requires them. Here, we're just using `sed`, which is really direct regex replacement. In my mind, it's almost more confusing to find + replace `{TEST_OSM}` vs just `TEST_OSM`?

Reply:
I just wanted it for readability (to connote that `TEST_OSM` would be overwritten). Looking around maybe a double `{{}}` would be best? (single curly braces designate associative arrays).